lms/static/sass/features/_course-upgrade-message.scss

```diff
@@ -43,6 +41,7 @@ Search for the courseware_verified_certificate_upsell promotion ID.
 
 .section.section-upgrade p {
   margin: 0.5em 0;
+  display: inline-block;
```

Comment:
Currently, in the course home sidebar upgrade the upgrade button is to the right of the cert icon.  
I see in your screenshot it is below even without a discount. Not sure if it is because of these css changes but we should probably keep it to the right at least when there is no discount?

Reply:
Hm. Yeah, that makes sense. I'll tweak it.